ci/prow/config.yaml

```diff
@@ -242,7 +242,6 @@ presubmits:
         command:
         - "/coverage"
         args:
-        - "--postsubmit-gcs-bucket=knative-prow"
         - "--postsubmit-job-name=post-knative-serving-go-coverage"
         - "--artifacts=$(ARTIFACTS)"
         - "--profile-name=coverage_profile.txt"
```

Comment:
This is default, please remove too. Here and everywhere.

Reply:
Done.